test/pull/out/.keboola/manifest.json

```diff
@@ -83,22 +83,22 @@
     {
       "branchId": %%TEST_BRANCH_BAR_ID%%,
       "componentId": "keboola.variables",
-      "id": "%%TEST_BRANCH_BAR_CONFIG_PYTHON_TRANSFORMATION_SHARED_CODE_VARIABLES_ID%%",
-      "path": "other/keboola.variables/python-transformation-shared-code-variables",
+      "id": "%%TEST_BRANCH_BAR_CONFIG_SHARED_CODE_VARIABLES_ID%%",
+      "path": "other/keboola.variables/shared-code-variables",
```

Comment:
Zmena poradia sposobena zmenou `path` (v testoch sa zoradzuje podla path, aby to bolo stale rovnake).